R/chol_solve_linear_system.R

```diff
@@ -0,0 +1,7 @@
+# solves Ax = b for x
```

Comment:
For consistency, I'd use roxygen style function comments everywhere, even if you're not exporting them.

Reply:
Actually, I think this sort of comment is OK as long as it is deliberate and meant only for developers (and not end users).